x-pack/test/functional/screenshots/baseline/security/reset_session_page/_reset_session_page.ts

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../ftr_provider_context';
```

Comment:
I noticed some of the imports were off by a level here and a few other places. You may want to double check these.
import { FtrProviderContext } from '../../../../ftr_provider_context';
import { WebElementWrapper } from '../../../../../../../test/functional/services/lib/web_element_wrapper';

Reply:
@jeramysoucy Thank you for the review.

- I Edited the file path for the `ftr_provider_context` and the `test/functional/services/lib/web_element_wrapper`. 

- And I changed the location of the files and edited the `--updateBaselines` options.

Reply:
`--updateBaselines` is the command; I shouldn't have made changes for this. I revert this.

Reply:
`--updateBaselines` is an option to the `functional_test_runner` script. When you're ready to run your test you can use the `functional_tests_server` script to start a test server, then use the `functional_test_runner` script in a separate terminal to run your tests.
Something like this...
Terminal 1: `node scripts/functional_tests_server.js --config [path to desired config.ts]`
Terminal 2: `node scripts/functional_test_runner --config [path to desired config.ts]`
Just add `--updateBaselines` when calling the test runner when you want to update the baseline images.

Reply:
Ok ✅